modules/spring-config/src/main/resources/META-INF/mule-module.xsd

```diff
@@ -150,24 +150,42 @@
                         </xsd:documentation>
                     </xsd:annotation>
                 </xsd:attribute>
-                <xsd:attribute name="namespace" use="required">
+                <xsd:attribute name="category" type="categoryType" default="COMMUNITY">
                     <xsd:annotation>
                         <xsd:documentation>
-                            Expected namespace of the module to look for when generating the schemas.
+                            Expected category of the module.
                         </xsd:documentation>
                     </xsd:annotation>
                 </xsd:attribute>
-                <xsd:attribute name="category" type="categoryType" default="COMMUNITY">
+                <xsd:attribute name="vendor" type="xsd:string" default="MuleSoft">
                     <xsd:annotation>
                         <xsd:documentation>
-                            Expected category of the module.
+                            Expected vendor of the module.
```

Comment:
From the jdoc:

`
  /**
   * Returns the name of the extension's vendor
   * This name is used to:
   * <ul>
   * <li>Represent the extension's vendor</li>
   * <li>Differentiate different extensions with the same name, to give the possibility of having two connectors
   * with the same name, for example, one made by MuleSoft and the second one by a third party</li>
   * </ul>
   *
   * @return the name of the extension's vendor
   */`